docs/README.md

```diff
@@ -1,115 +1,188 @@
+
+    To use this option run the following command:
+
+    ```bash
+    sudo ./scripts/setup_build_env.sh
+    ```
 
     b) The following method is universal and should work for most Linux OS distributions.
 
-    - MTL: Follow the [MTL setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/build.md) for installation.
+    - XDP-tools with eBpf: Follow the simple guide [XDP-tools](https://github.com/xdp-project/xdp-tools.git) for installation instructions.
+    - libfabric: Follow the [libfabric from source](https://github.com/ofiwg/libfabric?tab=readme-ov-file#building-and-installing-libfabric-from-source) for installation instructions.
+    - MTL: Follow the [MTL setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/build.md) for installation instructions.
+    - E810 driver: Follow the [MTL NIC setup guide](https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/doc/e810.md) for installation instructions.
     - gRPC: Refer to the [gRPC documentation](https://grpc.io/docs/languages/cpp/quickstart/) for installation instructions.
     - Install required packages:
+
         - Ubuntu/Debian
         ```bash
         sudo apt-get update
-        sudo apt-get install libbsd-dev
+        sudo apt-get install libbsd-dev cmake make libibverbs-dev librdmacm-dev dracut
         ```
         - Centos stream
         ```bash
-        sudo yum install -y libbsd-devel
+        sudo yum install -y libbsd-devel cmake make libibverbs-devel librdmacm-devel dracut
         ```
+
     - Install the irdma driver and libfabric
+
     ```bash
     ./scripts/setup_rdma_env.sh install
     ```
+
     - Reboot
 
-2. **Clone the repository**
-   ```sh
+4. **Build the Media Proxy binary**
+
+    To build Media Communications Mesh and make SDK available for development, run:
+
+    ```bash
+    $ ./build.sh
+    ```
+
+    By following these instructions, you'll be able to perform the basic build and installation of the Media Communications Mesh application.
+
+### Basic usage
+
+The program "media_proxy" and SDK library will be installed on system, after the "build.sh" script run successfully. To run "Media Proxy" execute below command:
+
+```bash
+$ media_proxy
+INFO: TCP Server listening on 0.0.0.0:8002
+INFO: gRPC Server listening on 0.0.0.0:8001
+```
+
+This will start the media proxy in blocking mode and confirm that build was successful. To close it, press `Ctrl+C`
+
+### Dockerfiles build
+
+#### Prerequisites
+
+- Linux server (Intel Xeon processor recommended, ex. Sapphire Rapids).
+- Network Interface Card (NIC) compatible with DPDK (ex. Intel Ethernet Controller E810-C).
+- (recommended) Update NIC drivers and firmware, more info and latest drivers [Support](#support)
+- Docker engine (recommended with Buildx toolkit) configured and installed.
+
+#### Build the Docker images
+
+1. **Clone the repository**
```

Comment:
Is this step not redundant? (the repository is there already.)

However, if you want to show the whole path from scratch, then we miss here:
$ sudo ./scripts/setup_build_env.sh
$ ./build.sh